handbook/communication/content_guidelines/style_and_mechanics.md

```diff
@@ -722,7 +722,12 @@ Only use _we_ and _our_ (as in “our GitHub integration”) in informal documen
 
 ### Feature names
 
-Don’t capitalize descriptive product or feature names. Features are not proper nouns. Avoid phrasing that can imply official product names.
+When referencing the product or category, the following should be treated as proper, singular nouns:
+
+- Universal Code Search
+- Batch Changes
```

Comment:
I don't like the idea that we're making this different case-by-case. It now says that "descriptive" product or feature names should not be capitalized, but it's not clear to me how "batch changes" is less "descriptive" than "code insights", "code monitoring", "code intelligence", and our other features? I'd really prefer to know that I can either refer to both "Batch Changes" and "Code Insights" or to both "batch changes" and "code insights" the same way, respectively.

Reply:
@felixfbecker - agreed. This was meant as a short term solution to support the Batch Changes launch. Marketing will eventually be updating our style guide for all products and features, but we need to get alignment with product and engineering first. At this point, we wanted to start with Batch Changes since there was a hard deadline.